content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -239,7 +239,7 @@ Full <abbr>parameters</abbr> (key and value) are listed separately under
   hand or with the command `dvc config --local`.
 
 - `.dvc/cache`: The <abbr>cache</abbr> directory will store your data in a
-  special [structure](#structure-of-the-cache-directory). The data files and
+  special [structure](/doc/user-guide/glossary/dvc-cache). The data files and
```

Comment:
Good! There's another 10+ instances of this link/anchor throughout docs though... Let's grep em all

Reply:
OK, done. Where it was possible, tried to preserve style by retaining the anchor link.

Reply:
All previous links to the anchor should still use the anchor in the new location. Just linking to the cache concept page could be redundant if the doc already has a cache tooltip, for example. Let's assume link to `#structure-of-the-cache-directory` for a good reason, thanks.

Reply:
> All previous links to the anchor should still use the anchor in the new location.

OK, revisited and checked these again... all with original anchor should have it now.